intake_solr/source.py

```diff
@@ -64,24 +73,57 @@ def __init__(self, query, base_url, core, qargs=None, metadata=None,
         super(SOLRSequenceSource, self).__init__(metadata=metadata)
 
     def _get_schema(self):
-        return base.Schema(datashape=None,
-                           dtype=None,
-                           shape=None,
-                           npartitions=1,
-                           extra_metadata={})
+        """Do a 0 row query and get the number of hits from the response"""
+        qargs = self.qargs.copy()
+        qargs["rows"] = 0
+        start = qargs.get("start", 0)
```

Comment:
The user may want to start the query at a different position, so we take that into account.

Reply:
:scroll: If we add support for Cursors, then we can't use the `start` option, according to SOLR documentation.

Reply:
I don't know what people would normally use.
Does offsetting with start cause the server to scan the whole table, or is solr smart here?